morpheus/_lib/include/morpheus/messages/memory/inference_memory.hpp

```diff
@@ -24,14 +24,32 @@
 
 namespace morpheus {
 /**
- * TODO(Documentation)
+ * @addtogroup messages
+ * @{
+ * @file
  */
 
 #pragma GCC visibility push(default)
+/**
+ * @brief This is a base container class for data that will be used for inference stages. This class is designed to
+    hold generic tensor data in cupy arrays
```

Comment:
On the Python side they are cupy arrays, on the C++ side they are Morpheus `TensorObject`s which are built on top of RMM device buffers.

Reply:
got it

Reply:
updated comments